docs/WritingTests.asciidoc

```diff
@@ -299,6 +299,32 @@ WORKER_CLASS = desktop
 # WORKER_CLASS is not set
 --------------------------------------------------------------------------------
 
+=== Automatic retries of jobs
+
+You might encounter flaky openQA tests that fail sporadically. While the best
```

Comment:
The ""While" implies there's a 2nd part within the same sentence but there isn't.